include/HubFramework/HUBViewControllerScrollHandler.h

```diff
@@ -78,6 +78,17 @@
 - (UIEdgeInsets)contentInsetsForViewController:(HUBViewController *)viewController
                          proposedContentInsets:(UIEdgeInsets)proposedContentInsets;
 
+
+/**
+ *  Return the center point of an overlay coponent used in a view controller
+ *
+ *  @param viewController The view controller in question
+ *  @param proposedCenterPoint The center point that the Hub Framework is proposing
+ *
+ */
+- (CGPoint)centerPointForOverlayComponentInViewController:(HUBViewController *)viewController
```

Comment:
@JohnSundell I want to center an overlay component when the keyboard is visible. Probably this approach is not the best.

 I've just looked at the code once more again. I'm not sure if the following function in `HubViewController` works just like it should:
```
- (CGPoint)overlayComponentCenterPoint
{
    CGRect frame = self.view.bounds;
    frame.origin.y = self.collectionView.contentInset.top;
    frame.size.height -= self.visibleKeyboardHeight + CGRectGetMinY(frame);

    CGPoint proposedCenterPoint = CGPointMake(CGRectGetMidX(frame), CGRectGetMidY(frame));
    return [self.scrollHandler centerPointForOverlayComponentInViewController:self
                                                          proposedCenterPoint:proposedCenterPoint];
}
```
It doesn't compute correctly middle point if the keyboard is visible. Removing following lines would sort this problem because view automatically changes its size if the keyboard is displayed. 
```
    frame.origin.y = self.collectionView.contentInset.top;
    frame.size.height -= self.visibleKeyboardHeight + CGRectGetMinY(frame);
```